mojave/items/guns/weapons/rifle.dm

```diff
@@ -148,6 +149,7 @@
 	bolt_type = BOLT_TYPE_NO_BOLT
 	empty_alarm = TRUE
 	semi_auto = TRUE
+	can_bayonet = TRUE
```

Comment:
We'll have to talk more about bayonets, but this is kind of a slippery slope and if we add them to one weapon we should add them to more. There are balance concerns regarding bayonets that I believe Original has already brought up.

Reply:
i had previously removed this so i'm not sure why it's still here (likely due to reverting shitstorm)